css-three-column-layout/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <section>
+    <div class="row">
+      <div class="column-full">
+        <h2>Designed for life science. Optimized for SaMD and Digital Health.</h2>
+      </div>
+    </div>
+  </section>
+  <section>
+    <div class="row">
+      <div class="column-one-third-image">
+        <img src="images/merge.png" alt="Merge">
+      </div>
+      <div class="column-one-third-image">
+        <img src="images/cloud-computing.png" alt="Cloud Computing">
+      </div>
+      <div class="column-one-third-image">
+        <img src="images/discussion.png" alt="Discussion">
+      </div>
+    </div>
+  </section>
+  <section>
+    <div class="row">
+      <div class="column-one-third">
+        <h3>Streamlined QMS</h3>
+      </div>
+      <div class="column-one-third">
+        <h3>Integrates with existing tools</h3>
+      </div>
+      <div class="column-one-third">
+        <h3>In-depth Onboarding</h3>
+      </div>
+    </div>
+  </section>
+  <section>
+    <div class="row">
+      <div class="column-one-third">
+        <p>A QMS that includes modules for all stages of the product life
+          cycle from Design Control to CAPA
+        </p>
+      </div>
+      <div class="column-one-third">
+        <p>
+          Enzyme QMS integrates with your existing tools, such as JIRA, GitHub
+          and Jenkins with Enzyme
+        </p>
+      </div>
+      <div class="column-one-third">
+        <p>
+          We train your entire team and provide you with support when and as it's needed
+        </p>
+      </div>
+    </div>
```

Comment:
Try reorganizing your HTML so that you only have 1 row of 3 columns, each containing the image, heading, and paragraph.